src/libfuncs/uint64.rs

```diff
@@ -1 +1,112 @@
+use super::{LibfuncBuilder, LibfuncHelper};
+use crate::{metadata::MetadataStorage, types::TypeBuilder};
+use cairo_lang_sierra::{
+    extensions::{
+        int::unsigned::{Uint64Concrete, Uint64Traits, UintConcrete, UintConstConcreteLibfunc},
+        lib_func::SignatureOnlyConcreteLibfunc,
+        GenericLibfunc, GenericType,
+    },
+    program_registry::ProgramRegistry,
+};
+use melior::{
+    dialect::arith::{self, CmpiPredicate},
+    ir::{attribute::IntegerAttribute, Block, Location, Value},
+    Context,
+};
 
+pub fn build<'ctx, 'this, TType, TLibfunc>(
+    context: &'ctx Context,
+    registry: &ProgramRegistry<TType, TLibfunc>,
+    entry: &'this Block<'ctx>,
+    location: Location<'ctx>,
+    helper: &LibfuncHelper<'ctx, 'this>,
+    metadata: &mut MetadataStorage,
+    selector: &Uint64Concrete,
+) -> Result<(), std::convert::Infallible>
+where
+    TType: GenericType,
+    TLibfunc: GenericLibfunc,
+    <TType as GenericType>::Concrete: TypeBuilder,
+    <TLibfunc as GenericLibfunc>::Concrete: LibfuncBuilder,
+{
+    match selector {
+        UintConcrete::Const(info) => {
+            build_const(context, registry, entry, location, helper, metadata, info)
+        }
+        UintConcrete::Operation(_) => todo!(),
+        UintConcrete::SquareRoot(_) => todo!(),
+        UintConcrete::Equal(info) => build_equal(context, registry, entry, location, helper, info),
+        UintConcrete::ToFelt252(_) => todo!(),
+        UintConcrete::FromFelt252(_) => todo!(),
+        UintConcrete::IsZero(_) => todo!(),
+        UintConcrete::Divmod(_) => todo!(),
+        UintConcrete::WideMul(_) => todo!(),
+    }
+}
+
+pub fn build_const<'ctx, 'this, TType, TLibfunc>(
+    context: &'ctx Context,
+    registry: &ProgramRegistry<TType, TLibfunc>,
+    entry: &'this Block<'ctx>,
+    location: Location<'ctx>,
+    helper: &LibfuncHelper<'ctx, 'this>,
+    metadata: &mut MetadataStorage,
+    info: &UintConstConcreteLibfunc<Uint64Traits>,
+) -> Result<(), std::convert::Infallible>
+where
+    TType: GenericType,
+    TLibfunc: GenericLibfunc,
+    <TType as GenericType>::Concrete: TypeBuilder,
+    <TLibfunc as GenericLibfunc>::Concrete: LibfuncBuilder,
+{
+    let value = info.c;
+    let value_ty = registry
+        .get_type(&info.signature.branch_signatures[0].vars[0].ty)
+        .unwrap()
+        .build(context, helper, registry, metadata)
+        .unwrap();
+
+    let op0 = entry.append_operation(arith::constant(
+        context,
+        IntegerAttribute::new(value.try_into().unwrap(), value_ty).into(),
```

Comment:
This will fail when the constant value is greater than `isize::MAX`. I think we should use the format string here.
```suggestion
        Attribute::parse(context, &format!("{value} : {value_ty}").unwrap(),
```